### ISSUE TITLE
Quickfix for CA1727: Use PascalCase for named placeholders in the logging message template

### ISSUE THREAD

**Issue by Bosch-Eli-Black** [dotnet/roslyn-analyzers#5727]
### Analyzer

**Diagnostic ID**: [CA1727](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1727) (Note, the documentation for this analyzer doesn't appear to exist).

### Describe the improvement

CA1727 is triggered for code such as this:
```c#
_logger.LogError("Encountered unknown status code {statusCode}", message.Status);
```

It'd be great if there were a quickfix (Ctrl+.) available to fix CA1727. 

### Describe suggestions on how to achieve the rule

Examples:
```
{statusCode} -> {StatusCode}
{status_code} -> {StatusCode}
{statuscode} -> {Statuscode} // Not great, but this is all we can do.
{status} -> {Status}
{0} // No fix available
```

### Additional context

After updating our code analyzers, we noticed quite a few places where CA1727 was triggered. Would have been great to be able to fix these with a quickfix! 🙂

**Comment by Matt-17** [comment]
I support this.
